source/core/ut_color_helper.pks

```diff
@@ -0,0 +1,11 @@
+create or replace package ut_color_helper as
```

Comment:
I propose to rename it to `ut_console_color_helper` or `ut_ansiconsole_coloring_helper` to proper reflect it's functionality